src/translations/en.json

```diff
@@ -652,7 +652,15 @@
     "HighTapRatio":"Ratio (high tap position)",
     "ImportRegulationRule":"Import regulation rule",
 
-    "GenerateTapRows" : "Generate tap rows",
+    "AddRows": "Add rows",
+    "DeleteRows": "Delete rows",
+    "MoveUpRows": "Move up rows",
+    "MoveDownRows": "Move down rows",
+
+    "AddTapRowsDialogTitle" : "How many rows do you want to add ?",
+    "AddTapRowsButton" : "Add",
```

Comment:
Well we need a unique generic translation key for "Add" instead of duplicate it for multiple usage but same value.

Reply:
I'm not sure about this, I mean depends how you use translation keys.
For me it's not supposed to be a basic translation but a fonctionnal message depending of where it's use in the code and in the UI that can be changed easily exactly where we want to.

I take these 3 lines in the file :
```
    "Execute": "Execute",
    "AddContingencyList": "Add",
    "DeleteContingencyList": "Delete",
```
For me the first line is not good, and the 2 next are the correct way but it's subjective. The problem is there the 2 way in the same translation file. Have you defined the good translation practice to follow somewhere ?

Reply:
Ok need to be defined by the team.